packages/clerk-js/src/ui/components/UserProfile/AccountPage.tsx

```diff
@@ -12,21 +12,22 @@ import { UserProfileSection } from './UserProfileSection';
 import { Web3Section } from './Web3Section';
 
 export const AccountPage = withCardStateProvider(() => {
-  const { attributes, saml, social } = useEnvironment().userSettings;
+  const { attributes, social } = useEnvironment().userSettings;
   const card = useCardState();
   const { user } = useUser();
 
   const showUsername = attributes.username.enabled;
   const showEmail = attributes.email_address.enabled;
   const showPhone = attributes.phone_number.enabled;
   const showConnectedAccounts = social && Object.values(social).filter(p => p.enabled).length > 0;
-  const showSamlAccounts = saml && saml.enabled && user && user.samlAccounts.length > 0;
```

Comment:
@NicolasLopes7 @izaaklauer Do you know what `saml.enabled` means in this case? Is it `true` when you have at least one SAML connection?